package.json

```diff
@@ -24,7 +25,8 @@
     "redux-logger": "^3.0.6",
     "redux-persist": "^6.0.0",
     "redux-saga": "^1.1.3",
-    "typescript": "~3.7.2"
+    "typescript": "^4.1.6",
+    "yarn": "^1.22.11"
```

Comment:
@alokesh985 Remove this and add it globally to your machine.

Reply:
Done.